test/test_pyrsh.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import time
+import unittest
+import tempfile
+import shutil
+import sys
+import MockSSH
+import paramiko
+from pyrsh import main
+
+from argparse import Namespace
+
+def recv_all(channel):
+    while not channel.recv_ready():
+        time.sleep(0.1)
+    stdout = ''
+    while channel.recv_ready():
+        stdout += channel.recv(1024)
+    return stdout
+
+
+class MockCiscoTestCase(unittest.TestCase):
```

Comment:
@GitSergeyGit `MockCiscoTest`, that sounds strong, but can we name it more accurate please?